README.md

```diff
@@ -1 +1,406 @@
 # fs-promise-util
+A utility library for file system interaction on *nix machines. This library utilizes the graceful-fs, an improvement over the fs module.
+
+## Requirements
+
+* `Node.js`: >= `v4.x`
+* `Platform`: `Darwin`, `Unix` or `Linux` (_Windows is not supported at this time_)
+
+## Installation
+
+```bash
+npm install fs-promise-util
+```
+
+## Usage
+
+This module exposes the following methods.
+
+## Methods in alphabetical order
+
+### fs-promise-util.appendFile (file, data, options)
+
+This method appends data to a file, creating the file if it does not exist. It returns a promise for fs.appendFile (https://nodejs.org/dist/latest-v7.x/docs/api/fs.html#fs_fs_appendfile_file_data_options_callback)
+
+
+* file [ string | Buffer | number ] filename or file descriptor
+* data [ string | Buffer ]
+* options [ Object | string ]
+
+	* encoding [ string | null ] default = 'utf8' 
+	* mode [ integer ] default = 0o666 
+	* flag [ string ] default = 'a' 
+                                                
+```javascript
+return await fs-promise-util
```

Comment:
Think of setting up your examples kind of like the following:

```javascript
import fs from 'fs-promise-util';

export async function saveMessage (message = '') {
  return await fs
    .appendFile(
      '/path/to/messages.log',
      message,
      { encoding : 'utf8' })
    .catch((err) => console.error(err));
}
```

A few things to note:

* indent with 2 spaces (this is counter to how we code the .js file, but this is because it displays well as a readme in GitHub)
* notice the example shows use of `import` and the code is wrapped in an `async` function